### DISCUSSION TITLE
Is it possible to crop?

### DISCUSSION THREAD

**Discussion by baek-sang** [mifi/lossless-cut#1847] [Support]
<img width="868" alt="crop" src="https://github.com/mifi/lossless-cut/assets/69100154/25e3f150-578e-4c15-8cd4-84c7de468cf8">

I want to select and cut only a portion of the video. 

ffmpeg crop option
```
-filter:v "crop=out_w:out_h:x:y"
```

**Comment by mifi** [comment]
See faq
